pyneuroml/swc/ExportNML.py

```diff
@@ -0,0 +1,535 @@
+import logging
+import tempfile
+from typing import Dict, List, Optional, Set
+
+import neuroml.writers as writers
```

Comment:
This looks good, well done. I think the file should be `test_ExportNML.py`, just for consistency, though.